src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/request/DeleteCaseTasksAction.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.request;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.io.Serializable;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+@ToString
+public class DeleteCaseTasksAction implements Serializable {
+
+    public static final long serialVersionUID = 432973322;
```

Comment:
Can an Unique SerialVersionID be created please ?

Reply:
Done